modules/programs/direnv.nix

```
@@ -160,6 +168,11 @@ in {
       )
     '');
 
+    programs.xonsh = mkIf cfg.enableXonshIntegration {
+      xonshrc = "xontrib load direnv";
+      extraPackages = ps: [ pkgs.xonsh.xontribs.xonsh-direnv ];
```

```suggestion
      extraPackages = [ config.programs.xonsh.finalPackage.xontribs.xonsh-direnv ];
```
Lists are properly coerced into functions :)
Also, should probably default to xontribs from the xonsh package.